pkg/option/config.go

```diff
@@ -216,8 +379,42 @@ func GetTunnelModes() string {
 	return fmt.Sprintf("%s, %s, %s", TunnelVXLAN, TunnelGeneve, TunnelDisabled)
 }
 
-// daemonConfig is the configuration used by Daemon.
-type daemonConfig struct {
+// getEnv returns the environment variable to be used for the given option name.
+func getEnv(option string) string {
```

Comment:
In other contexts "getenv" returns the value of the environment variable. Maybe rename this as `getEnvName()` instead?